integration-tests/testnet-scripts/start-chain.sh

```diff
@@ -9,7 +9,8 @@ BIN=$1
 #     mnemonic: "crackle snap pop ... etc",
 #     allocation: "10000000000stake,10000000000footoken",
 #     stake: "5000000000stake",
-#     number: "0",
+#     val_id: "1",
```

Comment:
"val_id" probably won't be a number in most cases. Something like "al" might be better to demonstrate.